### DISCUSSION TITLE
How to change EMBED_MODEL

### DISCUSSION THREAD

**Discussion by Sooloom** [open-webui/open-webui#795] [General]
I have modified the EMBED-MODEL configuration in backend/config.py and changed it to "m3e large". It seems that I did not download it, so I am not sure which directory to place this model in. I am requesting help.

**Comment by jannikstdl** [comment]
The parameter `EMBED_MODEL` at this time does nothing. Wait until #772 is merged. You can change to a different sentence transformer model in the Dockerfile then.

**Comment by antonsapt4** [comment]
Hi @jannikstdl https://github.com/open-webui/open-webui/pull/772 is merged right? to change the embed should we rebuild docker? or just the ENV on dockerfile and build -t would be enough?

And my current Ollama I already download `nomic-embed-text:latest` will the command still the same?

Edit: Oh and I'm using mac

**Reply by jannikstdl** [reply]
Hi, yes you need to rebuild the Dockerfile in the latest project. 

Try to git clone the latest project and open the Dockerfile in an Editor. There you can change the ENV to any sentence transformer embedding model. 

The standard is the same as before so change this to a different one. 

Then build the Docker Image and you are good to go!
